tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/multiz_performance.tsx

```diff
@@ -5,7 +5,7 @@ export const multiz_performance: Feature<number> = {
   category: 'GAMEPLAY',
   description: 'How detailed multi-z is. Lower this to improve performance',
   component: createDropdownInput({
-    [-1]: 'Highest',
+    [-1]: 'No Optimization',
```

Comment:
nooo dont use this jargon

Reply:
GOD DAMN IT MOTHBLOCKS IM BEING TOLD DIFFERENT THINGS

Reply:
Lemon's concerns were about the values, not the names-- the names I chose are the best from what I can tell

Reply:
Reverting to the names then